app/views/user_bets/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<% if current_user && @option %>
+  <%= render 'shared/bet_display', bet: @bet, option: @option %>
+
+  <hr>
+  <div class="row">
+    <div class="col col-lg-6 offset-lg-3">
+      <div class="card">
+        <div class="card-header text-muted text-center">
+          Enter your betting amount
+        </div>
+        <%= form_for [@bet, @user_bet] do |f| %>
+          <%= f.hidden_field :user_id, value: current_user.id %>
```

Comment:
Using the find_or_initialize above will make it so you dont have to specify the values for all of these fields